Description: don't fail if /dev/kmsg isn't set up
 It's not crucial, and containers can be set up with ro root which will
 prevent it from working anywa.
 Note - this is already in the github staging tree.
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Forwarded: no

Index: lxc/src/lxc/conf.c
===================================================================
--- lxc.orig/src/lxc/conf.c	2012-11-26 16:52:52.161010000 -0600
+++ lxc/src/lxc/conf.c	2012-11-26 16:57:58.287683598 -0600
@@ -2309,10 +2309,8 @@
 		return -1;
 	}
 
-	if (setup_kmsg(&lxc_conf->rootfs, &lxc_conf->console)) {
+	if (setup_kmsg(&lxc_conf->rootfs, &lxc_conf->console)) // don't fail
 		ERROR("failed to setup kmsg for '%s'", name);
-		return -1;
-	}
 
 	if (setup_tty(&lxc_conf->rootfs, &lxc_conf->tty_info, lxc_conf->ttydir)) {
 		ERROR("failed to setup the ttys for '%s'", name);
